stubs/networkx/@tests/stubtest_allowlist.txt

```diff
@@ -0,0 +1,21 @@
+# overloading a class-decorated method crashes stubtest
+networkx\.(algorithms\.)?(boundary\.)?edge_boundary
+networkx\.(algorithms\.)?(bridges\.)?local_bridges
+networkx\.(algorithms\.)?(clique\.)?node_clique_number
+networkx\.(algorithms\.)?(shortest_paths\.)?(generic\.)?shortest_path
+networkx\.(convert_matrix\.)?from_numpy_array
+networkx\.(convert_matrix\.)?from_pandas_adjacency
+networkx\.(convert_matrix\.)?from_pandas_edgelist
+networkx\.(generators\.)?(random_clustered\.)?random_clustered_graph
+networkx\.(relabel\.)?relabel_nodes
+
+# Stubtest doesn't understand aliases of class-decorated methods
```

Comment:
This may be the same issue as https://github.com/python/mypy/issues/6700
Where the alias will use the non-decorated version

Reply:
```suggestion
# Stubtest doesn't understand aliases of class-decorated methods (possibly https://github.com/python/mypy/issues/6700 )
```